HHS/Head-Start-TTADP
# [TTAHUB-3529] Monitoring goal card for RTR

## Description of change

This makes changes to the standard goal card when displaying a monitoring goal.

## How to test

- Review the code.
- Make sure all changes in the jira are in place when the goal is a monitoring goal.

## Issue(s)

* https://jira.acf.gov/browse/TTAHUB-3529


## Checklists

### Every PR



### Before merge to main


### Production Deploy


### After merge/deploy
